_content/site-config.json

```diff
@@ -3,5 +3,6 @@
   "site_tagline": "Informasi Faskes & Alkes untuk COVID-19",
   "site_description": "#WargaBantuWarga adalah situs yang memuat informasi seputar COVID-19 dan kontak fasilitas/alat kesehatan di seluruh Indonesia yang dikumpulkan relawan melalui pencarian di internet atau media sosial.",
   "site_url": "https://www.wargabantuwarga.com",
-  "whatsapp_contact_url": "https://bit.ly/hotlinewarga"
+  "whatsapp_contact_url": "https://bit.ly/hotlinewarga",
+  "discussion_url": "https://kcov.id/wbw-discuss"
```

Comment:
This data is coming from CMS Netlify.
You can not change it directly via codebase.

You need to change in `public/admin/config.yml` the update it via Netlify CMS.

Reply:
But for testing, you can go with it.

Reply:
Or if I can give you a suggestion, we have a hardcode static data in directory `constants`, if you don't wanna touch the Netlify CMS configuration, you can put the URL on that directory

Reply:
Waa, I'm sorry for that.

Reply:
Okay Mas, thanks for the suggestion. I will try it.